### PR TITLE
Schedule edits

### PR DESCRIPTION
**Pull request by AlbertDang46**

### CODE REVIEW

File: pages/SchedulePage.js [hunk 1]
```diff
@@ -38,14 +38,17 @@ export default class SchedulePage extends Component {
...
+            <TouchableOpacity style={styles.eventRow} activeOpacity={Res.workshopDetails[item.id] ? 0.2 : 1} onPress={() => this.handleRowPress(item)}>
                 <View style={styles.eventIconBox}>
                     <Icon style={styles.eventIcon} name='md-information-circle' size={24} color={'black'} />
                 </View>
                 <View style={styles.eventData}>
                     <Text style={styles.eventNameText}>{item.title}</Text>
                     <Text style={styles.eventTimeLocationText}>{item.time + " \u00B7 " + item.location}</Text>
                 </View>
+                <View style={styles.eventChevron}>
+                    {Res.workshopDetails[item.id] && <Icon name='ios-arrow-forward' size={24} color={'white'} visibility={'none'} />}
```
Comment (ChrispyLam) [inline pages/SchedulePage.js]: what does visibility none do

File: pages/SchedulePage.js [hunk 2]
```diff
@@ -131,6 +134,10 @@ const styles = StyleSheet.create({
         color: '#fefefe',
         opacity: 0.7
     },
+    eventChevron: {
+        flex: 1,
+        alignItems: 'center'
```
Comment (ChrispyLam) [inline pages/SchedulePage.js]: justifyContent center to vertically center the chevron with the row

File: pages/SchedulePage.js (hunk 1 above)
Comment (AlbertDang46) [reply pages/SchedulePage.js]: oh wait i forgot to take it out

Comment (AlbertDang46) [reply pages/SchedulePage.js]: it doesn't do anything

File: pages/SchedulePage.js (hunk 2 above)
Comment (terrencejames) [reply pages/SchedulePage.js]: oh true, it looked fine on my phone but if i look closer youre right

File: resources/strings.js [hunk 3]
```diff
@@ -399,9 +399,9 @@ const schedule = [
   {id: "crypticChaosIILunch", title: "Cryptic Chaos II", day: "2", time: "12:10pm - 12:55pm", location: "Fields"},
   {id: "lunchII", title: "Lunch II", day: "2", time: "1:00pm - 1:45pm", location: "Dining Hall"},
   {id: "crypticChaosILunch", title: "Cryptic Chaos I", day: "2", time: "1:00pm - 1:45pm", location: "Fields"},
-  {id: "workshopSessionIII", title: "Workshop Session III  >", day: "2", time: "1:55pm - 2:45pm", location: ""},
-  {id: "workshopSessionIV", title: "Workshop Session IV  >", day: "2", time: "2:55pm - 3:45pm", location: ""},
-  {id: "workshopSessionV", title: "Workshop Session V  >", day: "2", time: "3:55pm - 4:45pm", location: ""},
+  {id: "workshopSessionIII", title: "Workshop Session III", day: "2", time: "1:55pm - 2:45pm", location: ""},
+  {id: "workshopSessionIV", title: "Workshop Session IV", day: "2", time: "2:55pm - 3:45pm", location: ""},
+  {id: "workshopSessionV", title: "Workshop Session V", day: "2", time: "3:55pm - 4:45pm", location: ""},
```
Comment (ChrispyLam) [inline resources/strings.js]: Add "Various Locations" or something to location. Same with Day 2 workshop sessions

**Pull request merged**